terraform_files/baremetal/variables-libvirt.tf

```diff
@@ -174,3 +174,9 @@ variable "static_macs" {
   description = "If true, static macs are configured for the network interfaces"
   type        = bool
 }
+
+variable "single_node_ip" {
+  description = "IP address of single node.  Used for DNS"
+  type = string
+  default = "192.168.126.10"
```

Comment:
lets remove default and set it in the code, i want this part to fail in case nobody sets ip

Reply:
OK